Metalama.Framework.Engine/AssertionFailedException.cs

```diff
@@ -11,5 +11,7 @@ public sealed class AssertionFailedException : Exception
         public AssertionFailedException() { }
 
         public AssertionFailedException( string message ) : base( message ) { }
+
+        internal AssertionFailedException( in AssertionFailedInterpolatedStringHandler messageHandler ) : base( messageHandler.GetFormattedText() ) { }
```

Comment:
Nit: if a struct has a single reference type field, then I think it's pointless to use `in` with it.